data/json/monsters.json

```diff
@@ -2901,12 +2901,238 @@
+            [ "child_items", 65 ]
+        ]
+    },
+    "hp":30,
+    "death_function":["NORMAL"],
+    "special_attacks":[["LEAP", 10]],
+    "description":"This crouching child-mutant's face is dominated by a pair of huge black eyes and its fingertips end in sharp claws.",
+    "//": "GUILT because it still looks enough like a child to evoke pity",
+    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BLEED", "GUILT", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_shriekling",
+    "name": "shriekling",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"z",
+    "color":"magenta",
+    "size":"SMALL",
+    "material":"flesh",
+    "diff":3,
+    "aggression":100,
+    "morale":100,
+    "speed":95,
+    "melee_skill":2,
+    "melee_dice":2,
+    "melee_dice_sides":2,
+    "melee_cut":1,
+    "dodge":1,
+    "armor_bash":0,
+    "armor_cut":0,
+    "vision_day":30,
+    "vision_night":3,
+    "death_drops": {
+        "subtype": "collection",
+        "//": "cloth as any other zombie (always), additional items from child_items (sometimes)",
+        "groups": [
+            [ "default_zombie_clothes", 100 ],
+            [ "child_items", 65 ]
+        ]
+    },
+    "hp":20,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SHRIEK", 5]],
+    "description":"This heavily mutated child zombie twitches and flails its limbs in painful looking spasms as it runs about.",
+    "//": "GUILT because it still looks enough like a child to evoke pity",
+    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BLEED", "GUILT", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_creepy",
+    "name": "creepy crawler",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"z",
+    "color":"dkgray",
+    "size":"SMALL",
+    "material":"flesh",
+    "diff":3,
+    "aggression":80,
+    "morale":100,
+    "speed":55,
+    "melee_skill":2,
+    "melee_dice":2,
+    "melee_dice_sides":2,
+    "melee_cut":1,
+    "dodge":1,
+    "armor_bash":0,
+    "armor_cut":0,
+    "vision_day":3,
+    "vision_night":30,
+    "death_drops": {
+        "subtype": "collection",
+        "//": "cloth as any other zombie (always), additional items from child_items (sometimes)",
+        "groups": [
+            [ "default_zombie_clothes", 100 ],
+            [ "child_items", 65 ]
+        ]
+    },
+    "hp":25,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SHRIEK", 5]],
+    "description":"What was once a child is now a mutant beast with blackened skin and massive eyes.  This abomination's vile form makes mockery of its human origin.",
+    "//": "no GUILT because it no longer looks enough like a child to evoke pity",
+    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BLEED", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
```

Comment:
I assume it crawls on all fours rather than like crawling zed (just on arms).
Could get climbing, to make it a bit more insect-like.
Almost all of them shriek, maybe move the role of "night shrieker" to the waif one (it doesn't differ from shriekling much) and give this one, say, acid barf (the weak acid zed one, not spitter or corrosive one)?

The description could also mention it runs on all fours, currently only name and speed suggest that.